manifests/download.pp

```diff
@@ -0,0 +1,64 @@
+# == Definition: archive::download
```

Comment:
@igalic Maybe document this type as for compatibility with camptocamp?  You would otherwise always use the native `archive` type directly?